packages/fx-core/src/core/middleware/projectMigratorV3.ts

```diff
@@ -93,15 +92,16 @@ import { getLocalizedString } from "../../common/localizeUtils";
 const Constants = {
   vscodeProvisionBicepPath: "./templates/azure/provision.bicep",
   launchJsonPath: ".vscode/launch.json",
-  appYmlName: "app.yml",
-  appLocalYmlName: "app.local.yml",
+  appYmlName: "teamsapp.yml",
+  appLocalYmlName: "teamsapp.local.yml",
   tasksJsonPath: ".vscode/tasks.json",
   reportName: "migrationReport.md",
   envWriteOption: {
     // .env.{env} file might be already exist, use append mode (flag: a+)
     encoding: "utf8",
     flag: "a+",
   },
+  environmentFolder: "teamsAppEnv",
```

Comment:
I think this constant can be added to versionMetadata too. 
And as the env folder can be changed, defaultEnvironmentFolder could be better.

Reply:
moved